_schemaV2.graphql

```diff
@@ -18488,6 +18491,16 @@ enum sort {
   DESC
 }
 
+type SpecialistBio {
+  bio: String
+  email: String
+  firstName: String
+  image: Image
```

Comment:
**praise**: Really glad we're able to use our Image type here to enable clients to crop/resize as necessary. ⭐